js/components/markers/helpers.js

```diff
@@ -0,0 +1,41 @@
+const MAXSIZE = 2;
+
+/**
+ * getPosition return the position in pixels for the markers
+ *
+ * @param {*} duration duration in seconds of the asset
```

Comment:
should it be a `number`?

Reply:
Hahaha I just forget to add the type here.